src/geode/mesh/core/regular_grid.cpp

```diff
@@ -332,6 +334,38 @@ namespace geode
         return impl_->cell_attribute_manager();
     }
 
+    template < index_t dimension >
+    BoundingBox< dimension > RegularGrid< dimension >::bounding_box() const
+    {
+        BoundingBox< dimension > bbox;
+        bbox.add_point( this->origin() );
+        Point< dimension > extreme_point;
+        for( const auto d : Range{ dimension } )

```

Comment:
or first compute the Index and then use the method point( index ). The result will be the same